src/components/ProtestForm/ProtestForm.js

```diff
@@ -28,13 +28,11 @@ const OpeningText = () => {
   return (
     <div style={{ textAlign: 'center' }}>
       <h2>{t('opening.one')}</h2>
-      <h3>{t('opening.two')}</h3>
-      <div>{t('opening.three')}</div>
-      <a href="https://www.youtube.com/watch?v=uhaOAPDAXz4" role="button" tabIndex="1" target="_blank" rel="noopener noreferrer">
+      <p>{t('opening.two')}</p>
+      <p style={{ margin: 0 }}>{t('opening.three')}</p>
+      <a href="#0" role="button" tabIndex="1" onClick={() => window.open('https://www.youtube.com/watch?v=uhaOAPDAXz4')}>
```

Comment:
Why did you change the link form href to onclick? Also dropping opening.three was on purpose?

Reply:
This was how the link was implemented in the first place, so I just sticked to it.
Dropping opening.three was on purpose yes.

Reply:
Sorry, reviewing after a long day :see_no_evil: